src/framework/theme/components/stepper/stepper.spec.ts

```diff
@@ -1,8 +1,111 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import {
+  ComponentFixture,
+  fakeAsync,
+  TestBed,
+} from '@angular/core/testing';
 
 import { NbStepperComponent } from './stepper.component';
 import { NbStepComponent } from './step.component';
 import { NbIconModule } from '../icon/icon.module';
+import {
+  Component,
+  DebugElement,
+  NO_ERRORS_SCHEMA,
+} from '@angular/core';
+import { By } from '@angular/platform-browser';
+
+@Component({
+  selector: 'nb-step-changed-test',
+  template: `
+    <nb-stepper [selectedIndex]="1">
+      <nb-step [label]="labelOne">
+        <ng-template #labelOne>First step</ng-template>
+      </nb-step>
+
+      <nb-step [label]="labelTwo">
+        <ng-template #labelTwo>Seconds step</ng-template>
+      </nb-step>
+
+      <nb-step [label]="labelThree">
+        <ng-template #labelThree>First step</ng-template>
+      </nb-step>
+    </nb-stepper>
+  `,
+})
+
+export class NbStepChangeTestComponent {}
+
+describe('Stepper: Step Change', () => {
+  let stepper: NbStepperComponent;
+  let fixture: ComponentFixture<NbStepChangeTestComponent>;
+  let stepperEl: DebugElement;
+
+  beforeEach(fakeAsync (() => {
+    TestBed.configureTestingModule({
+      schemas: [NO_ERRORS_SCHEMA],
+      declarations: [NbStepChangeTestComponent, NbStepComponent, NbStepperComponent],
```

Comment:
I think it's better to import `NbStepperModule` instead of providing individual components. This way we also ensure module exports all needed components.